.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!-- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!-- Describe your changes in detail -->
+
+## Motivation and Context
+<!-- Why is this change required? What problem does it solve? -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## Types of Changes
+<!-- Please select all items that apply either now or after creating the pull request: -->
+- [ ] Documentation update
+- [ ] Bug fix (a change that fixes an issue)
+- [ ] New feature (a change which adds functionality)
+- [ ] Breaking change (the change breaks or has potential to break existing functionality)
+
+This pull request is based on
+- [ ] *master*, because it is a bug fix or an update to the documentation.
+- [ ] *develop*, because it introduces a new feature.
+
+
+## Checklist:
+<!-- Please select all items that apply either now or after creating the pull request. -->
+<!-- If you are unsure about any of these items, do not hesitate to ask! -->
+- [ ] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md).
+- [ ] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac/blob/master/ContributorAgreement.md).
+- [ ] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.txt).
+- [ ] My code follows the code style of this project (PEP8 with up to 100 characters per line).<sup>1</sup>
```

Comment:
We have a webpage describing our code style, right? I'd rather link to that.

Reply:
No, not really. That is more or less the full-extend of the style description. I'm not sure if we should elaborate on that.